### ISSUE TITLE
Support for setting Azure Front Door cache duration

### ISSUE THREAD

**Issue by oliwer** [hashicorp/terraform-provider-azurerm#11508]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Using the Azure console or CLI it is possible to set the cache duration of a forwarding rule. But there is no such option in the azurerm provider.

* azurerm_frontdoor in the `forwarding_configuration` block

### Potential Terraform Configuration

```hcl
resource "azurerm_frontdoor" "example" {
  name                                         = "example-FrontDoor"
  location                                     = "EastUS2"
  resource_group_name                          = azurerm_resource_group.example.name
  enforce_backend_pools_certificate_name_check = false

  routing_rule {
    name               = "exampleRoutingRule1"
    accepted_protocols = ["Http", "Https"]
    patterns_to_match  = ["/*"]
    frontend_endpoints = ["exampleFrontendEndpoint1"]
    forwarding_configuration {
      forwarding_protocol = "MatchRequest"
      backend_pool_name   = "exampleBackendBing"
      cache_enabled = true
      cache_use_dynamic_compression = true
      # Missing option :(
      # cache_duration = "5h"
    }
  }

  ...
}
```

### References

* https://docs.microsoft.com/en-us/cli/azure/afd/rule/action?view=azure-cli-latest#az_afd_rule_action_add
* https://docs.microsoft.com/en-us/azure/frontdoor/front-door-caching

**Comment by sebader** [comment]
will be closed by https://github.com/terraform-providers/terraform-provider-azurerm/pull/12831

**Comment by oliwer** [comment]
I have not tested since I do not work with Azure anymore, but I guess this issue can be closed since #12831 was merged.
